PSJira/Internal/ConvertTo-JiraWorklogitem.ps1

```diff
@@ -0,0 +1,92 @@
+function ConvertTo-JiraWorklogitem
```

Comment:
please capitalize the `i` in `item` (camel case)

_don't forget the file name_